### ISSUE TITLE
yarn run deploy fails

### ISSUE THREAD

**Issue by frogg** [OasysFoundation/Oasys-Toolbox-React#249]
```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/dotenv/lib/main.js:21

Read more here: http://bit.ly/2tRViJ9
```

**Comment by frogg** [comment]
https://github.com/motdotla/dotenv/issues/261

**Comment by frogg** [comment]
fixed in 929550308edeae6783690f6c12a8c7585bc15baf
